govtool/backend/src/VVA/Config.hs

```diff
@@ -82,6 +83,8 @@ data VVAConfigInternal
       , vVAConfigInternalSentrydsn              :: String
         -- | Sentry environment
       , vVAConfigInternalSentryEnv              :: String
+        -- | Metadata validation service enabled
+      , vVAConfigInternalMetadataValidationEnabled :: Bool
```

Comment:
I wasn't aware of this strange naming convention until now. Let's leave it as is but I think it is worth refactoring it since it's a leftover from the previous project codename.